# MVP de Engenharia de Dados

## Catarina Carneiro da Silva

Neste estudo o objetivo é analisar a influência de algumas características de um funcionário em seu salário, com foco no setor terciário (comércio, serviços e turismo). O dataset utilizado neste projeto são os microdados da RAIS 2022 (Relação Anual de Informações Sociais), proveniente do Ministério do Trabalho. Essa base de informações é muito abrangente, podendo originar diversas inferências. No entanto, para ter resultados mais objetivos com o desejado, utilizei um subconjunto da base original, considerando apenas as variáveis mais relevantes.

###Perguntas a serem respondidas:

1- A discriminação das mulheres pode ser comprovada por dados? Qual o salário médio das mulheres e dos homens?

2- A discriminação de raça pode ser comprovada por dados? Qual o salário médio de cada uma das raças analisadas?

3- Acredita-se que as pessoas mais novas costumam receber salários menores devido a menor experiencia, pode-se comprovar essa crença com dados?

4- Acredita-se que as pessoas com menor escolaridade costumam receber salários menores devido ao menor conhecimento, pode-se comprovar essa crença com dados?

5- São Paulo (código 35) é considerado um dos estados mais avançados do país e, por isso, com maior nivel de salários. Pode-se comprovar essa crença com dados?

6- Qual atividade do setor terciário apresenta o maior nível salarial?

### Carga - Camada Bronze

Fiz o download dos microdados da RAIS de 2022 no site do Ministério do Trabalho em um arquivo CSV e criei uma nova tabela no Databricks Comunity Edition com base nesse arquivo.

In [0]:
%sql
SELECT *
FROM rais2022

cnae faixa_etaria escolaridade idade cor sexo uf remuneracao 94120 7 1 61 8 1 50 2014.0 84116 5 1 30 99 1 50 1116.52 84116 5 1 31 99 1 50 1091.7 84116 4 1 28 99 1 50 1091.7 84116 8 1 69 99 1 50 2075.62 84116 4 1 26 99 1 50 2553.11 84116 4 1 28 99 1 50 2533.8 84116 7 1 50 99 1 50 1212.0 84116 5 1 33 99 1 50 2724.79 84116 4 1 25 99 1 50 2706.22 84116 7 1 59 99 1 50 3066.28 84116 4 1 29 99 1 50 1074.37 43991 4 1 26 1 1 50 1285.62 1512 8 1 70 4 1 50 0.0 1512 5 1 39 8 1 50 1356.67 2306 7 1 56 9 1 50 1229.4 43134 4 1 25 2 1 50 1483.85 10139 6 1 47 8 1 50 0.0 49302 6 1 46 8 1 50 1450.0 41204 7 1 57 8 1 50 0.0 85937 3 1 18 9 1 50 1246.0 94201 6 1 42 4 1 50 20673.17 1130 5 1 36 8 1 50 2425.48 94308 3 1 21 2 1 50 569.36 43215 6 1 44 9 1 50 1500.0 41204 6 1 44 8 1 50 0.0 71197 3 1 24 9 1 50 4864.16 56112 7 1 57 9 1 50 0.0 55108 5 1 32 2 1 50 1506.91 41204 7 1 55 8 1 50 0.0 49213 6 1 44 2 1 50 0.0 94201 7 1 58 2 1 50 0.0 2306 3 1 20 4 1 50 661.2 94201 7 1 59 2 1 50 18105.23 94201 7 1 50 8 1 50 0.0 1130 7 1 55 8 1 50 3318.97 47296 4 1 29 8 1 50 1978.38 1130 7 1 62 8 1 50 2197.25 47113 6 1 47 2 1 50 0.0 22293 5 1 36 9 1 50 3248.03 2306 5 1 35 4 1 50 661.2 47440 7 1 55 9 1 50 0.0 1512 6 1 43 8 1 50 1830.28 22293 5 1 31 2 1 50 1456.81 1130 7 1 50 8 1 50 2353.14 1512 8 1 76 8 1 50 1431.67 47440 7 1 57 6 1 50 1212.0 96033 3 1 22 8 1 50 1533.78 1130 7 1 60 8 1 50 2312.8 1130 6 1 40 8 1 50 2425.26 38394 6 1 45 2 1 50 905.33 49248 6 1 49 1 1 50 0.0 94201 7 1 53 4 1 50 19601.22 38327 5 1 31 9 1 50 0.0 1130 6 1 44 8 1 50 2779.41 81290 7 1 53 8 1 50 1240.36 43991 6 1 48 9 1 50 0.0 71120 3 1 18 2 1 50 0.0 10112 5 1 33 2 1 50 1319.07 45307 7 1 56 8 1 50 1782.22 47440 6 1 47 8 1 50 2100.91 10139 6 1 40 8 1 50 0.0 82997 4 1 28 2 1 50 1620.0 94201 4 1 26 8 1 50 14955.74 55108 6 1 42 9 1 50 1643.44 94936 3 1 18 4 1 50 854.04 1610 7 1 61 4 1 50 1402.5 10112 7 1 54 4 1 50 4428.78 94201 6 1 40 8 1 50 0.0 2306 4 1 25 9 1 50 661.2 45200 5 1 33 9 1 50 1782.84 2306 5 1 37 8 1 50 661.2 2306 5 1 31 2 1 50 10468.92 85996 6 1 46 8 1 50 1212.0 94910 7 1 62 4 1 50 1163.52 47440 3 1 19 9 1 50 1440.92 2306 7 1 50 8 1 50 661.2 81303 7 1 53 8 1 50 1440.01 46320 5 1 31 9 1 50 1764.05 94201 5 1 39 2 1 50 0.0 94201 6 1 47 4 1 50 0.0 55108 7 1 52 2 1 50 1542.4 46818 6 1 45 9 1 50 0.0 2306 7 1 57 4 1 50 661.2 2306 6 1 46 8 1 50 0.0 2306 4 1 26 9 1 50 661.2 45307 7 1 51 8 1 50 1353.75 94201 8 1 65 2 1 50 4730.48 1130 7 1 59 8 1 50 0.0 1130 7 1 50 8 1 50 0.0 1130 7 1 55 8 1 50 0.0 94201 4 1 26 8 1 50 18862.88 2101 6 1 47 9 1 50 972.97 2306 6 1 47 9 1 50 661.2 68226 5 1 34 2 1 50 1323.82 47890 7 1 61 8 1 50 0.0 1512 7 1 63 8 1 50 1675.59 94201 4 1 29 8 1 50 16607.97 1512 6 1 42 8 1 50 1366.28 94201 6 1 42 1 1 50 0.0 94201 8 1 67 8 1 50 65223.32 1130 7 1 61 2 1 50 2002.96 81117 7 1 50 8 1 50 1212.09 1512 6 1 48 2 1 50 1291.67 1415 7 1 59 8 1 50 21207.52 1512 7 1 54 8 1 50 1318.38 46320 5 1 36 9 1 50 1910.16 2306 6 1 44 9 1 50 934.34 49302 7 1 63 2 1 50 0.0 2306 7 1 58 2 1 50 801.2 1512 7 1 53 2 1 50 1309.37 2101 6 1 45 9 1 50 1322.22 94201 7 1 58 4 1 50 0.0 49302 7 1 52 9 1 50 0.0 46915 6 1 48 2 1 50 1756.22 1113 6 1 49 2 1 50 0.0 1512 7 1 61 8 1 50 0.0 1512 5 1 31 8 1 50 2000.0 1512 3 1 23 8 1 50 1279.03 68102 7 1 59 9 1 50 1317.64 1512 6 1 43 9 1 50 2055.56 2306 4 1 29 2 1 50 661.2 1130 8 1 72 8 1 50 0.0 2306 3 1 24 9 1 50 661.2 1130 5 1 34 8 1 50 2218.8 1130 5 1 32 8 1 50 2192.44 2306 5 1 34 2 1 50 778.8 94201 7 1 51 2 1 50 0.0 59111 5 1 31 8 1 50 1332.5 55108 4 1 28 8 1 50 1401.66 1512 5 1 36 2 1 50 1234.44 47890 7 1 59 2 1 50 2746.09 1512 8 1 68 2 1 50 0.0 1512 6 1 46 8 1 50 1366.28 1512 7 1 57 8 1 50 1553.36 85139 8 1 76 8 1 50 0.0 43991 5 1 33 8 1 50 0.0 1512 8 1 68 8 1 50 2440.07 1512 6 1 48 2 1 50 1363.39 1512 7 1 54 9 1 50 1197.69 82997 6 1 41 2 1 50 1620.0 1512 7 1 59 8 1 50 2468.07 23303 7 1 55 9 1 50 1334.0 1512 7 1 61 8 1 50 2020.99 47440 7 1 59 8 1 50 1456.81 1512 5 1 37 2 1 50 0.0 1610 6 1 49 9 1 50 1350.0 41204 6 1 44 8 1 50 1615.0 94995 7 1 53 8 1 50 0.0 1512 5 1 

###Catálogo de dados:

CNAE - CNAE da atividade do emprego segundo a classificação 2.0 com a segmentação por classe. Neste momento encontram-se todas as CNAES registradas na RAIS, no entanto para analisar somente o setor terciário, como pretendido, terei que filtrar apenas as classes entre 45111 e 69206 na próxima etapa.

Como são 673 classes registradas, deixarei para especificar cada uma na próxima etapa, quando os dados já estiverem filtrados para o setor desejado.

In [0]:
%sql
SELECT DISTINCT cnae
FROM rais2022
ORDER BY cnae DESC;

cnae 99008 97005 96092 96033 96025 96017 95291 95215 95126 95118 94995 94936 94928 94910 94308 94201 94120 94111 93298 93212 93191 93131 93123 93115 92003 91031 91023 91015 90035 90027 90019 88006 87301 87204 87123 87115 86909 86607 86500 86402 86305 86224 86216 86101 85996 85937 85929 85911 85503 85422 85414 85333 85325 85317 85201 85139 85121 85112 84302 84256 84248 84230 84221 84213 84132 84124 84116 82997 82920 82911 82300 82202 82199 82113 81303 81290 81222 81214 81125 81117 80307 80200 80129 80111 79902 79121 79112 78302 78205 78108 77403 77390 77331 77322 77314 77292 77233 77225 77217 77195 77110 75001 74901 74200 74102 73203 73190 73122 73114 72207 72100 71201 71197 71120 71111 70204 69206 69125 69117 68226 68218 68102 66304 66291 66223 66215 66193 66134 66126 66118 65502 65421 65413 65308 65201 65120 65111 64999 64930 64921 64913 64701 64638 64620 64611 64506 64409 64387 64379 64361 64352 64344 64336 64328 64310 64247 64239 64221 64212 64107 63992 63917 63194 63119 62091 62040 62031 62023 62015 61906 61434 61426 61418 61302 61205 61108 60225 60217 60101 59201 59146 59138 59120 59111 58298 58239 58221 58212 58191 58131 58123 58115 56201 56121 56112 55906 55108 53202 53105 52508 52401 52397 52320 52311 52290 52231 52222 52214 52125 52117 51307 51200 51129 51111 50998 50912 50301 50220 50211 50122 50114 49507 49400 49302 49299 49248 49230 49221 49213 49124 49116 47890 47857 47849 47831 47822 47814 47741 47733 47725 47717 47636 47628 47610 47598 47571 47563 47555 47547 47539 47521 47512 47440 47431 47423 47415 47326 47318 47296 47245 47237 47229 47211 47130 47121 47113 46931 46923 46915 46893 46877 46869 46851 46842 46834 46826 46818 46796 46745 46737 46729 46711 46699 46656 46648 46630 46621 46613 46524 46516 46494 46478 46460 46451 46443 46435 46427 46419 46397 46371 46362 46354 46346 46338 46320 46311 46231 46222 46214 46192 46184 46176 46168 46150 46141 46133 46125 46117 45439 45421 45412 45307 45200 45129 45111 43991 43916 43304 43291 43223 43215 43193 43134 43126 43118 42995 42928 42910 42235 42227 42219 42138 42120 42111 41204 41107 39005 38394 38327 38319 38220 38211 38122 38114 37029 37011 36006 35301 35204 35140 35131 35123 35115 33295 33210 33198 33171 33163 33155 33147 33139 33121 33112 32990 32922 32914 32507 32400 32302 32205 32124 32116 31047 31039 31021 31012 30997 30920 30911 30504 30423 30415 30326 30318 30121 30113 29506 29492 29450 29441 29433 29425 29417 29301 29204 29107 28691 28666 28658 28640 28631 28623 28615 28542 28534 28526 28518 28402 28330 28321 28313 28291 28259 28241 28232 28224 28216 28151 28143 28135 28127 28119 27902 27597 27511 27406 27333 27325 27317 27228 27210 27104 26809 26701 26604 26523 26515 26400 26329 26311 26221 26213 26108 25993 25934 25926 25918 25501 25438 25420 25411 25390 25322 25314 25225 25217 25136 25128 25110 24521 24512 24491 24431 24423 24415 24393 24318 24245 24237 24229 24211 24121 24113 23991 23923 23915 23494 23427 23419 23303 23206 23192 23125 23117 22293 22234 22226 22218 22196 22129 22111 21238 21220 21211 21106 20991 20941 20932 20924 20916 20738 20720 20711 20631 20622 20614 20525 20517 20401 20339 20321 20312 20291 20223 20215 20193 20142 20134 20126 20118 19322 19314 19225 19217 19101 18300 18229 18211 18130 18121 18113 17494 17427 17419 17338 17320 17311 17222 17214 17109 16293 16234 16226 16218 16102 15408 15394 15335 15327 15319 15297 15211 15106 14223 14215 14142 14134 14126 14118 13596 13545 13537 13529 13511 13405 13308 13235 13227 13219 13146 13138 13120 13111 12204 12107 11224 11216 11135 11127 11119 10996 10961 10953 10945 10937 10929 10911 10821 10813 10724 10716 10694 10660 10651 10643 10635 10627 10619 10538 10520 10511 10431 10422 10414 10333 10325 10317 10201 10139 10121 10112 9904 9106 8991 8932 8924 8916 8100 7294 7251 7243 7235 7227 7219 7103 6000 5003 3221 3213 3124 3116 2306 2209 2101 1709 1636 1628 1610 1598 1555 1547 1539 1521 1512 1423 1415 1393 1351 1342 1334 1326 1318 1229 1211 1199 1164 1156 1148 1130 1121 1113 999 977

Faixa_etaria - Faixa de idade do funcionário, valores esperados:

1-entre 10 a 14 anos/

2-entre 15 a 17 anos/

3-entre 18 a 24 anos/

4-entre 25 a 29 anos/

5-entre 30 a 39 anos/

6-entre 40 a 49 anos/

7-entre 50 a 64 anos/

8-65 anos ou mais.

Na base de dados também encontrei a categoria 99, que são os dados informados como ignorados. Na próxima etapa precisarei filtrar essas informações.

In [0]:
%sql
SELECT DISTINCT faixa_etaria
FROM rais2022
ORDER BY faixa_etaria DESC;

faixa_etaria 99 8 7 6 5 4 3 2 1

Escolaridade - Escolaridade do funcionário, valores esperados:

1-Analfabeto

2-Até 5ª Incompleto

3-5ª Completo Fundamental

4-6ª a 9ª Fundamental

5-Fundamental Completo

6-Médio Incompleto

7-Médio Completo

8-Superior Incompleto

9-Superior Completo

10-Mestrado

11-Doutorado

Na base de dados também encontrei a categoria 99, que são os dados informados como ignorados. Na próxima etapa precisarei filtrar essas informações.

In [0]:
%sql
SELECT DISTINCT escolaridade
FROM rais2022
ORDER BY escolaridade DESC;

escolaridade 99 11 10 9 8 7 6 5 4 3 2 1

Idade - Idade do funcionário. Por conter valores próximos de 100 e idades abaixo dos 18 anos, essa série causa estranhamento e gera necessidade de atenção ao se trabalhar com esses dados.

In [0]:
%sql
SELECT DISTINCT idade
FROM rais2022
ORDER BY idade DESC;

idade 100 99 98 97 96 95 94 93 92 91 90 89 88 87 86 85 84 83 82 81 80 79 78 77 76 75 74 73 72 71 70 69 68 67 66 65 64 63 62 61 60 59 58 57 56 55 54 53 52 51 50 49 48 47 46 45 44 43 42 41 40 39 38 37 36 35 34 33 32 31 30 29 28 27 26 25 24 23 22 21 20 19 18 17 16 15 14 13 12 11 0

Cor - Raça/cor segundo funcionário, valores esperados:

1-Indígena

2- Branca

4- Preta

6- Amarela

8- Parda

9- Não informado

Na base de dados também encontrei a categoria 99, que são os dados informados como ignorados. Na próxima etapa precisarei filtrar essas informações assim como a categoria 9 (não informado).

In [0]:
%sql
SELECT DISTINCT cor
FROM rais2022
ORDER BY cor DESC;

cor 99 9 8 6 4 2 1

Sexo - Gênero segundo funcionário, valores esperados:

1- Masculino

2- Feminino

9- Ignorado

Na próxima etapa precisarei filtrar as informações da categoria 9, que são os dados informados como ignorados.

In [0]:
%sql
SELECT DISTINCT sexo
FROM rais2022
ORDER BY sexo DESC;

sexo 9 2 1

UF - 26 estados brasileiros e o Distrito Federal, valores esperados:
11 - Rondônia /	12 - Acre /	13 - Amazonas /	14 - Roraima /	15 - Para /	16 - Amapa /	17 - Tocantins /	21 - Maranhão /	22 - Piaui /	23 - Ceará /	24 - Rio Grande do Norte /	25 - Paraíba /	26 - Pernambuco /	27 - Alagoas /	28 - Sergipe /	29 - Bahia /	31 - Minas Gerais /	32 - Espírito Santo	/ 33 - Rio de Janeiro /	35 - São Paulo /	41 - Paraná /	42 - Santa Catarina /	43 - Rio Grande do Sul /	50 - Mato Grosso do Sul /	51 - Mato Grosso /	52 - Goiás /	53 - Distrito Federal

Na base de dados também encontrei a categoria 99, que são os dados informados como ignorados. Na próxima etapa precisarei filtrar essas informações.

In [0]:
%sql
SELECT DISTINCT uf
FROM rais2022
ORDER BY uf DESC;

uf 99 53 52 51 50 43 42 41 35 33 32 31 29 28 27 26 25 24 23 22 21 17 16 15 14 13 12 11

remuneração - Remuneração média dos funcionários

In [0]:
%sql
SELECT DISTINCT remuneracao
FROM rais2022
ORDER BY remuneracao DESC;

remuneracao 3755863.0 3700000.08 3568937.82 3543183.0 3520000.0 3510114.6 3500000.07 3480000.0 3474116.26 3381833.6 3353097.57 3270748.0 3264000.0 3241680.72 3226753.64 3204797.74 3169920.0 3162030.09 3119000.0 3107000.0 3075000.0 3055000.0 3050370.69 3040000.0 2992781.52 2988356.91 2970364.03 2950233.77 2948900.62 2940000.0 2938370.0 2933294.0 2903321.0 2892751.34 2868000.0 2865163.55 2863278.21 2840652.98 2836471.74 2834632.66 2833329.6 2823955.47 2822148.21 2812403.02 2812182.52 2807372.21 2803372.0 2792094.62 2778073.68 2759886.91 2758000.0 2737258.13 2732070.12 2719548.08 2714926.35 2711638.42 2709016.0 2708146.57 2701929.37 2698589.2 2694818.68 2686393.77 2677915.71 2673182.4 2671735.29 2641722.02 2641633.46 2590626.64 2582890.0 2572853.96 2569668.49 2566409.66 2560000.0 2553837.28 2553498.22 2546398.17 2540255.64 2537250.18 2535664.03 2532000.0 2522628.0 2515220.8 2510959.98 2501894.05 2500549.8 2500000.0 2487343.66 2479583.66 2478431.77 2470187.94 2464473.98 2459605.04 2450601.45 2445588.01 2445040.0 2432000.0 2419003.68 2418089.07 2417979.03 2410851.42 2410546.8 2405794.9 2404800.0 2394963.72 2391925.46 2383222.74 2378240.75 2367083.33 2366910.09 2365275.96 2356714.55 2351155.06 2350840.0 2350575.09 2349009.62 2347223.55 2346040.0 2345932.73 2345786.76 2343286.55 2343260.0 2343158.48 2340580.0 2336407.42 2330719.4 2327979.05 2321103.14 2317488.6 2317157.9 2316264.0 2315848.0 2313434.82 2306286.81 2305200.0 2305000.0 2300789.0 2296319.0 2294720.58 2289993.72 2283618.72 2280000.0 2278462.12 2265639.75 2262398.24 2261608.16 2249657.38 2247510.0 2247000.0 2243200.0 2234784.23 2233689.0 2224419.33 2218541.52 2217750.0 2216757.34 2203224.77 2198622.8 2198614.78 2198361.6 2198002.84 2192062.45 2191200.0 2190804.02 2188206.35 2186424.0 2185729.18 2184793.94 2182680.0 2178381.38 2176391.77 2172535.65 2170503.15 2162130.94 2160455.11 2157989.78 2149754.16 2141670.0 2141520.0 2138357.74 2138161.0 2137360.0 2136090.0 2131851.87 2130774.6 2129969.48 2126649.85 2124427.0 2123259.8 2101580.17 2101406.0 2100845.83 2100143.66 2099060.0 2097987.67 2097265.0 2093496.36 2090755.13 2081442.36 2078000.0 2076132.0 2070138.68 2060000.0 2059415.0 2059367.92 2059083.0 2057240.0 2055869.7 2054248.12 2053135.93 2049653.55 2040000.0 2039972.0 2039893.2 2035995.85 2035032.0 2034117.75 2031000.0 2030000.0 2028126.66 2021087.87 2020752.0 2016399.9 2015568.4 2014444.29 2014422.0 2009407.64 2009244.46 2009124.49 2003647.33 2000000.37 1997086.08 1990888.55 1989836.14 1989526.74 1988173.34 1986284.79 1985676.71 1979771.96 1975000.0 1973397.09 1970000.0 1968835.0 1968454.86 1966000.0 1965461.24 1964156.42 1960643.23 1960563.14 1954869.4 1952758.0 1950291.0 1940919.0 1940622.95 1940212.1 1937723.2 1936217.35 1935750.03 1935000.0 1934351.14 1933092.92 1930000.0 1925195.0 1924871.14 1923000.0 1921666.57 1920000.0 1919721.83 1918562.66 1913590.52 1912516.0 1912201.0 1911475.99 1911370.44 1911183.96 1910464.77 1908884.3 1907582.44 1907503.24 1905165.8 1903447.73 1901820.0 1901718.0 1900198.56 1900000.08 1899800.0 1899218.8 1895897.22 1892083.72 1891413.28 1890750.29 1886737.63 1884986.63 1881509.73 1879797.79 1877050.78 1875000.0 1872940.16 1870000.0 1867948.99 1866510.0 1865971.3 1864817.1 1863358.72 1857650.41 1855686.49 1848935.52 1848340.0 1848190.0 1847317.32 1843814.41 1842840.98 1841765.09 1839338.49 1837158.97 1834432.03 1832665.0 1828562.41 1826548.38 1825379.13 1825198.74 1823519.71 1812045.4 1811470.83 1809504.6 1808406.29 1808390.57 1807961.93 1806984.41 1806007.44 1802630.55 1801735.41 1800400.0 1799300.0 1799000.0 1797969.7 1795927.04 1793663.0 1791214.0 1784234.1 1784105.2 1781907.11 1781686.01 1780472.76 1777928.8 1777810.4 1775810.11 1775760.0 1775307.63 1773662.08 1771646.33 1769675.91 1769285.06 1764085.38 1763479.38 1762637.0 1761901.82 1761900.0 1761899.17 1760869.6 1760348.88 1758468.92 1757815.61 1756914.27 1750000.0 1749380.98 1747192.07 1746229.63 1744940.16 1738340.0 1737581.85 1736735.91 1734966.59 1734490.65 1730673.35 1730238.0 17

###Transformação - Camada Silver
Para facilitar a consulta e limpar a base de dados farei as transformações necessárias.
Primeiro deletarei a coluna idade, já que os dados parecem estranhos e na coluna faixa_etaria os valores já aparecem agrupados, o que facilita a análise e oferece maior credibilidade ao resultado.

Para trabalhar apenas o setor terciário, como pretendido, filtrarei a coluna cnae para conter apenas as CNAES desse setor (entre as classes 45111 e 69206).

Para limpar o dataset, excluirei os seguintes dados:
o gênero que não corresponde ao sexo Feminino ou Masculino,  faixa etária e escolaridade com resultados ignorados, a cor com informações diferentes das tradicionais e as uf ignoradas.

In [0]:
%sql

CREATE OR REPLACE TABLE rais2022_silver AS
SELECT cnae, escolaridade, faixa_etaria, cor, sexo, remuneracao, uf
FROM rais2022
WHERE cnae >= 45111 AND cnae <= 69206
  AND sexo!=9
  AND faixa_etaria !=99
  AND escolaridade !=99
  AND cor!=9
  AND cor!=99
  AND uf!=99;

num_affected_rows num_inserted_rows

In [0]:
%sql
SELECT DISTINCT cnae
FROM rais2022_silver
ORDER BY cnae DESC;

cnae 69206 69125 69117 68226 68218 68102 66304 66291 66223 66215 66193 66134 66126 66118 65502 65421 65413 65308 65201 65120 65111 64999 64930 64921 64913 64701 64638 64620 64611 64506 64409 64387 64379 64361 64352 64344 64336 64328 64310 64247 64239 64221 64212 64107 63992 63917 63194 63119 62091 62040 62031 62023 62015 61906 61434 61426 61418 61302 61205 61108 60225 60217 60101 59201 59146 59138 59120 59111 58298 58239 58221 58212 58191 58131 58123 58115 56201 56121 56112 55906 55108 53202 53105 52508 52401 52397 52320 52311 52290 52231 52222 52214 52125 52117 51307 51200 51129 51111 50998 50912 50301 50220 50211 50122 50114 49507 49400 49302 49299 49248 49230 49221 49213 49124 49116 47890 47857 47849 47831 47822 47814 47741 47733 47725 47717 47636 47628 47610 47598 47571 47563 47555 47547 47539 47521 47512 47440 47431 47423 47415 47326 47318 47296 47245 47237 47229 47211 47130 47121 47113 46931 46923 46915 46893 46877 46869 46851 46842 46834 46826 46818 46796 46745 46737 46729 46711 46699 46656 46648 46630 46621 46613 46524 46516 46494 46478 46460 46451 46443 46435 46427 46419 46397 46371 46362 46354 46346 46338 46320 46311 46231 46222 46214 46192 46184 46176 46168 46150 46141 46133 46125 46117 45439 45421 45412 45307 45200 45129 45111

A série de cnaes foi filtrada e, com dados mais limpos, acrescentei o dicionário, com a atividade ligada a cada classe. Nas séries online do Ministério do Trabalho fiz o download em formato CSV dos nomes das atividades relacionadas a cada classe. Criei uma nova tabela no Databricks Comunity Edition com essas informações.

In [0]:
%sql
SELECT *
FROM classes

Classe Nome 69206 Atividades de Contabilidade, Consultoria e Auditoria Contabil e Tributaria 69125 Cartorios 69117 Atividades Juridicas, Exceto Cartorios 68226 Gestao e Administracao da Propriedade Imobiliaria 68218 Intermediacao na Compra, Venda e Aluguel de Imoveis 68102 Atividades Imobiliarias de Imoveis Proprios 66304 Atividades de Administracao de Fundos por Contrato ou Comissao 66291 Atividades Auxiliares dos Seguros, da Previdencia Complementar e dos Planos de Saude nao Especificadas Anteriormente 66223 Corretores e Agentes de Seguros, de Planos de Previdencia Complementar e de Saude 66215 Avaliacao de Riscos e Perdas 66193 Atividades Auxiliares dos Servicos Financeiros nao Especificadas Anteriormente 66134 Administracao de Cartoes de Credito 66126 Atividades de Intermediarios em Transacoes de Titulos, Valores Mobiliarios e Mercadorias 66118 Administracao de Bolsas e Mercados de Balcao Organizados 65502 Planos de Saude 65421 Previdencia Complementar Aberta 65413 Previdencia Complementar Fechada 65308 Resseguros 65201 Seguros-Saude 65120 Seguros Nao-Vida 65111 Seguros de Vida 64999 Outras Atividades de Servicos Financeiros nao Especificadas Anteriormente 64930 Administracao de Consorcios para Aquisicao de Bens e Direitos 64921 Securitizacao de Creditos 64913 Sociedades de Fomento Mercantil - Factoring 64701 Fundos de Investimento 64638 Outras Sociedades de Participacao, Exceto Holdings 64620 Holdings de Instituicoes Nao-Financeiras 64611 Holdings de Instituicoes Financeiras 64506 Sociedades de Capitalizacao 64409 Arrendamento Mercantil 64387 Bancos de Cambio e Outras Instituicoes de Intermediacao Nao-Monetaria 64379 Sociedades de Credito ao Microempreendedor 64361 Sociedades de Credito, Financiamento e Investimento - Financeiras 64352 Credito Imobiliario 64344 Agencias de Fomento 64336 Bancos de Desenvolvimento 64328 Bancos de Investimento 64310 Bancos Multiplos, sem Carteira Comercial 64247 Credito Cooperativo 64239 Caixas Economicas 64221 Bancos Multiplos, com Carteira Comercial 64212 Bancos Comerciais 64107 Banco Central 63992 Outras Atividades de Prestacao de Servicos de Informacao nao Especificadas Anteriormente 63917 Agencias de Noticias 63194 Portais, Provedores de Conteudo e Outros Servicos de Informacao na Internet 63119 Tratamento de Dados, Provedores de Servicos de Aplicacao e Servicos de Hospedagem na Internet 62091 Suporte Tecnico, Manutencao e Outros Servicos em Tecnologia da Informacao 62040 Consultoria em Tecnologia da Informacao 62031 Desenvolvimento e Licenciamento de Programas de Computador Nao-Customizaveis 62023 Desenvolvimento e Licenciamento de Programas de Computador Customizaveis 62015 Desenvolvimento de Programas de Computador Sob Encomenda 61906 Outras Atividades de Telecomunicacoes 61434 Operadoras de Televisao por Assinatura por Satelite 61426 Operadoras de Televisao por Assinatura por Microondas 61418 Operadoras de Televisao por Assinatura por Cabo 61302 Telecomunicacoes por Satelite 61205 Telecomunicacoes sem Fio 61108 Telecomunicacoes por Fio 60225 Programadoras e Atividades Relacionadas a Televisao por Assinatura 60217 Atividades de Televisao Aberta 60101 Atividades de Radio 59201 Atividades de Gravacao de Som e de Edicao de Musica 59146 Atividades de Exibicao Cinematografica 59138 Distribuicao Cinematografica, de Video e de Programas de Televisao 59120 Atividades de Pos-Producao Cinematografica, de Videos e de Programas de Televisao 59111 Atividades de Producao Cinematografica, de Videos e de Programas de Televisao 58298 Edicao Integrada a Impressao de Cadastros, Listas e de Outros Produtos Graficos 58239 Edicao Integrada a Impressao de Revistas 58221 Edicao Integrada a Impressao de Jornais 58212 Edicao Integrada a Impressao de Livros 58191 Edicao de Cadastros, Listas e de Outros Produtos Graficos 58131 Edicao de Revistas 58123 Edicao de Jornais 58115 Edicao de Livros 56201 Servicos de Catering, Bufe e Outros Servicos de Comida Preparada 56121 Servicos Ambulantes de Alimentacao 56112 Res


Seguem abaixo as novas séries, com as transformações realizadas conforme desejado.

In [0]:
%sql
SELECT DISTINCT sexo
FROM rais2022_silver
ORDER BY sexo DESC;

sexo 2 1

In [0]:
%sql
SELECT DISTINCT faixa_etaria
FROM rais2022_silver
ORDER BY faixa_etaria DESC;

faixa_etaria 8 7 6 5 4 3 2 1

In [0]:
%sql
SELECT DISTINCT escolaridade
FROM rais2022_silver
ORDER BY escolaridade DESC;

escolaridade 11 10 9 8 7 6 5 4 3 2 1

In [0]:
%sql
SELECT DISTINCT cor
FROM rais2022_silver
ORDER BY cor DESC;

cor 8 6 4 2 1

In [0]:
%sql
SELECT DISTINCT uf
FROM rais2022_silver
ORDER BY uf DESC;

uf 53 52 51 50 43 42 41 35 33 32 31 29 28 27 26 25 24 23 22 21 17 16 15 14 13 12 11

#####1- A discriminação das mulheres pode ser comprovada por dados? Qual o salário médio das mulheres e dos homens?

In [0]:
%sql
SELECT sexo, AVG(remuneracao) AS remuneracao_sexo
FROM rais2022_silver
GROUP BY sexo
ORDER BY remuneracao_sexo DESC;

sexo remuneracao_sexo 1 4339.0324753202785 2 3482.809441164602

In [0]:
%sql
SELECT sexo, COUNT(*) AS genero
FROM rais2022_silver
GROUP BY sexo
ORDER BY genero DESC;

sexo genero 1 7992716 2 6055556

O salário médio das mulheres do setor terciário foi de R$ 3.483 em 2022, o que representa 80% da remuneração média dos homens no setor e confirma o menor nível salarial do gênero feminino. Deve-se considerar que elas representam menos da metade (43,1%) dos trabalhadores do Brasil e é necessário pensar em formas de incentivar o aumento da entrada delas no mercado de trabalho.

#####2- A discriminação de raça pode ser comprovada por dados? Qual o salário médio de cada uma das raças analisadas?

In [0]:
%sql
SELECT cor, AVG(remuneracao) AS remuneracao_cor
FROM rais2022_silver
GROUP BY cor
ORDER BY remuneracao_cor DESC;

cor remuneracao_cor 6 6769.0480832367 2 4775.8470701512815 1 3711.0810999702894 4 3035.58368649696 8 3019.519925042666

In [0]:
%sql
SELECT cor, COUNT(*) AS raca
FROM rais2022_silver
GROUP BY cor
ORDER BY raca DESC;

cor raca 2 7334474 8 5630126 4 940329 6 116415 1 26928

A maior média salarial corresponde a quem se considera "6- amarelo" (R$ 6.769), seguida pela população "2- branca" (R$ 4.776) e as duas últimas categorias são das pessoas denominadas "4- pretas" (R$ 3.036) e "8- pardas" (R$ 3.020).

Apesar da população branca não ter as maiores remunerações, eles representam mais da metade (52,2%) dos trabalhadores brasileiros, confirmando a preferência do mercado de trabalho por esses funcionários. 

#####3- Acredita-se que as pessoas mais novas costumam receber salários menores devido a menor experiencia, pode-se comprovar essa crença com dados?

In [0]:
%sql
SELECT faixa_etaria, AVG(remuneracao) AS remuneracao_idade
FROM rais2022_silver
GROUP BY faixa_etaria
ORDER BY remuneracao_idade DESC;

faixa_etaria remuneracao_idade 6 5198.012395385265 8 5170.564066844556 7 4910.34251181779 5 4441.567947496604 4 2961.2288557839524 3 1969.1549368812493 2 963.4823195818942 1 638.910453759723

In [0]:
%sql
SELECT faixa_etaria, COUNT(*) AS idade
FROM rais2022_silver
GROUP BY faixa_etaria
ORDER BY idade DESC;

faixa_etaria idade 5 4066205 6 2997927 3 2461234 4 2309966 7 1918714 8 175003 2 116909 1 2314

Os colaboradores abaixo de 24 anos são os que recebem menores salários, confirmando a hipótese inicial da menor idade ser acompanhada por menor salário, enquanto aqueles entre 40 e 49 anos recebem em média os maiores valores.

Pode-se observar um preferência do mercado de trabalho por contratar pessoas entre 30 e 49 anos, somando esses dois grupos encontra-se 50,3% dos funcionários brasileiros. Sendo funcionário mais experientes, no entanto não tão velhos para se tornarem improdutivos.

#####4- Acredita-se que as pessoas com menor escolaridade costumam receber salários menores devido ao menor conhecimento, pode-se comprovar essa crença com dados?

In [0]:
%sql
SELECT escolaridade, AVG(remuneracao) AS remuneracao_escolaridade
FROM rais2022_silver
GROUP BY escolaridade
ORDER BY remuneracao_escolaridade DESC;

escolaridade remuneracao_escolaridade 10 26114.70055458715 11 18217.16102560787 9 11494.694465088623 8 4576.643150751637 7 2489.364135697803 5 2152.43010877633 4 2098.56872691009 1 2094.6398189387737 3 2047.5979745397283 6 2043.379161377498 2 1997.7664139095205

In [0]:
%sql
SELECT escolaridade, COUNT(*) AS formacao
FROM rais2022_silver
GROUP BY escolaridade
ORDER BY formacao DESC;

escolaridade formacao 7 8335590 9 2160002 6 992437 5 940002 8 792522 4 458750 3 163706 2 143959 10 28111 1 25461 11 7732

Os dados comprovam que há uma grande diferença salarial dependendo do grau de estudo do funcionário, com as pessoas com escolaridade até o 5º ano incompleto recebendo os menores salários (R$ 1.998). É interessante observar que o nível de renda de quem possui mestrado (R$ 26.115) é acima de quem tem doutorado (R$ 18.217), provalvemente por terem menos opções de carreira especificamente voltada para os doutorando.

Em relação a quantidade de pessoas empregadas, a maior parte possui o ensino médio completo (59,3%), seguida pela população com o nível superior completo (15,4%).

#####5- São Paulo (código 35) é considerado um dos estados mais avançados do país e, por isso, com maior nivel de salários. Pode-se comprovar essa crença com dados?

In [0]:
%sql
SELECT uf, AVG(remuneracao) AS remuneracao_uf
FROM rais2022_silver
GROUP BY uf
ORDER BY remuneracao_uf DESC;

uf remuneracao_uf 53 7151.892257216753 35 4815.875911582386 43 4088.623444486743 33 4015.8067896248176 42 3901.0926059527487 41 3767.8149652978695 51 3594.150289306611 26 3274.479465692664 31 3208.2392055226446 52 3190.5695646853187 15 3164.7026946137244 32 3155.698713634867 50 3146.0271781815673 12 3085.704951759378 13 3070.2781393959367 29 3017.7357424260963 11 3008.8920849372407 17 2953.38753866811 28 2952.1308295638687 23 2864.5434667901623 25 2837.3287468862513 21 2789.0826510185584 14 2663.5237636386523 27 2659.054198850918 22 2647.0274148921567 16 2634.673925574833 24 2619.702876531594

In [0]:
%sql
SELECT uf, COUNT(*) AS estado
FROM rais2022_silver
GROUP BY uf
ORDER BY estado DESC;

uf estado 35 4560038 31 1382166 33 1323783 41 897924 43 859083 42 618979 29 547159 52 420845 23 415390 26 410045 32 322358 53 317732 15 307198 51 266119 21 211613 50 171836 13 164022 24 145519 25 137697 22 125341 27 116801 11 95082 28 88432 17 63295 16 31226 12 24668 14 23921

Os dados mostram que a localidade com maior nível salarial é o Distrito Federal (R$ 7.152), retirando o protagonismo do sudeste, no entanto, compatível por ser uma área política, ficando São Paulo (R$ 4.816) em segundo lugar. Apesar de refutarmos a hipótese desse estado ter o maior rendimento, São Paulo compreende a maior parte dos trabalhadores brasileiros (32,5%).

#####6- Qual atividade do setor terciário apresenta o maior nível salarial?

In [0]:
%sql
SELECT cnae, AVG(remuneracao) AS remuneracao_cnae
FROM rais2022_silver
GROUP BY cnae
ORDER BY remuneracao_cnae DESC;

cnae remuneracao_cnae 64239 44606.74423299517 64336 35301.324690454385 64328 26944.89764044943 64611 24730.25076690215 64344 21592.638803582846 51307 21231.015 65308 19520.698817204302 64999 18691.44293066313 49400 18576.17177103448 64212 18203.867016377895 64409 18007.287166123788 64221 17927.9622254634 50301 15751.304219673155 64310 14904.374147066104 65413 14508.47700618616 61302 14130.614616600795 66304 13841.072682097762 64352 13696.661497377201 59138 13247.105323910479 50114 12987.761227495896 65120 12805.623581081074 50122 12801.793482517487 66118 12594.632354892214 64247 12472.85566648623 61434 12374.899864864867 65421 12310.35269540693 66126 11723.689501222527 64620 10865.221561404898 64361 10690.294864495252 61205 10140.258198418836 51200 10088.669785008715 52397 9971.386616415408 64506 9839.293662822634 51111 9361.034007883154 65111 9268.783052877377 46516 9133.988443068092 63917 8554.8842215256 46648 8498.866748597073 60217 8483.286288597053 62040 8434.187751695947 62015 8238.815474318691 65201 8208.154279847897 46818 8197.607536628248 64638 7999.970269685042 46524 7969.361080839648 49124 7711.88374368831 46214 7679.106772124847 62031 7585.25188909172 52320 7583.532542582082 51129 7485.920129620737 46362 7390.961743983258 63119 7341.103255877976 53105 7244.2997303409675 66193 7223.943737389043 62091 7057.362851897736 46842 7005.881829941962 46125 6872.668177150194 46621 6866.768108874403 46184 6832.111610563509 62023 6829.154318923375 50998 6731.819928335172 52311 6618.45234808588 66134 6577.332345010558 46443 6511.081663879801 58115 6480.881699945546 46699 6410.110332664296 60225 6291.138252252253 46460 6160.895247017551 46834 6144.635827879831 64921 6129.315552193648 49116 6069.357057282831 46613 5970.167164855582 64930 5964.083334462609 63194 5895.819439171524 46630 5738.653689135057 46222 5732.195419833626 46451 5629.499412705617 46826 5563.072114650672 65502 5450.647983250513 52401 5450.604519119352 58212 5420.679762367981 46923 5337.741209694128 66291 5196.996587618049 46141 4949.715776009092 61418 4934.2902888938825 59201 4877.962419642855 46150 4802.961160292221 59111 4790.938402689674 61108 4718.2350631146555 45111 4710.477658546389 47512 4692.546028590757 46231 4669.459021731617 46311 4660.54707878457 58239 4563.358615221989 50211 4505.796793721968 46893 4465.267032689533 58123 4464.576480550411 66223 4438.652866453796 58131 4319.5350430839 46851 4263.346071107478 58221 4118.592569081406 68102 4059.7988109321363 52508 4045.177253931116 46656 3932.031424799206 66215 3826.193325537886 46737 3819.039560306862 47717 3788.5603145127284 46931 3778.1529884429187 45129 3759.8335099884425 46168 3732.865114719379 64913 3710.342422879176 46354 3677.026421752873 47539 3622.722565966026 52117 3596.009359080761 46435 3557.0989676448444 46745 3540.3725249258964 46320 3529.5385120565293 69117 3465.7457633380045 68226 3439.7923199706593 64387 3436.6549999999997 46729 3405.768442006155 46371 3387.1746382765323 61906 3378.824447469708 64379 3340.2554754098364 69125 3300.601875833672 52214 3280.773827733499 47831 3279.498196922885 46478 3238.1975415714214 46192 3222.010136507296 64107 3205.574 50912 3159.1675603864733 49507 3137.7663867684473 46346 3076.399703196856 59120 3072.6160839160834 47130 3059.111119499389 46494 3055.044759792822 46117 3050.68360013624 68218 3044.143615029896 69206 3029.5053919983934 46796 3028.7206335662304 47733 3023.2506253139118 60101 2930.7554694178893 49302 2896.1764137837918 46397 2888.4871208286877 45412 2863.6276654992494 46915 2817.1632161935213 63992 2795.494181091877 47415 2793.2963452881804 47636 2791.1635290657655 46427 2780.067919221284 49221 2755.471018250645 47423 2744.4129249568578 49213 2730.962286369783 47610 2721.391993993699 52290 2713.410311944938 58298 2709.753041016754 46869 2693.555199999999 45307 2589.289792969767 47725 2581.7613353169295 50220 2566.9114822006463 49299 2537.665075903424 47113 2508.4286465228834 47547 2480.7774992379377 52231 2468.5838712740833 46419 2455.06

In [0]:
%sql
SELECT cnae, COUNT(*) AS classe
FROM rais2022_silver
GROUP BY cnae
ORDER BY classe DESC;

cnae classe 47113 1245976 56112 976229 49302 890320 47440 512026 47814 505605 47717 470442 45307 372989 49213 339547 47890 327442 47121 318698 47318 276377 64221 267733 56201 266816 69206 258598 55108 249036 47211 236321 47296 188943 45111 183779 45200 181820 47547 173867 46397 144229 47539 140489 47822 136565 62015 119951 47130 119214 62091 115481 61108 107582 65502 104242 53105 103909 62040 98323 46354 97109 46915 92852 47555 90810 64239 89504 47725 88728 64247 86821 52117 85549 49221 82956 46443 82530 47610 80582 62031 79435 69125 77219 69117 77148 47229 72860 47512 70932 47636 69704 46338 68231 49299 68047 62023 66505 47741 66372 47598 66175 47245 65583 46494 64486 46931 64030 63119 61756 47521 60969 61906 59499 45412 58248 46231 57888 46460 57000 52508 55137 52125 54281 66223 53240 46371 51849 68226 51811 52231 51598 68218 51178 47423 50411 63194 49730 64620 49654 46796 48519 47237 46849 46346 45013 51111 43130 60217 42717 46877 42389 52401 42287 46451 41887 46834 39577 52311 39496 47849 38877 53202 38425 68102 37870 46893 36036 46613 35522 47415 33989 66193 33007 49116 32453 46630 31956 47831 31718 46427 31642 64999 31109 49124 30697 65120 29896 52214 29733 47431 29323 46320 27454 46729 26977 46699 26904 60101 25734 46818 25595 47571 23827 61205 23274 46478 22311 46516 22281 46869 22025 46419 20671 66126 20450 50301 20312 49248 19547 66134 18469 49230 18383 52290 18016 46222 16709 47733 15928 46711 14964 46192 14871 55906 14757 46923 14483 46842 14301 46737 14078 65111 13276 46826 12839 46851 12235 64930 11807 63992 11265 46524 11195 58115 11018 46311 10992 59146 10913 46184 10754 46621 10581 64638 10160 50211 9366 46176 9196 47326 9112 46656 9089 46648 8910 61418 8896 58298 8655 52320 8102 64361 8044 58221 8034 59111 7882 45439 7541 58212 7196 46214 7017 65413 6951 50114 6721 56121 6684 52222 6664 46435 6645 65201 6575 47857 6347 51129 6249 66304 6178 45129 6057 46362 5734 66291 4765 50912 4554 64212 4213 64310 4039 46133 3948 58123 3779 46745 3711 47563 3671 49400 3625 46141 3518 64336 3279 64913 3112 46117 2936 46168 2833 65421 2482 66118 2412 46125 2337 46150 2327 58131 2205 66215 2138 64352 2097 58191 2075 64921 1983 64506 1821 50998 1814 52397 1791 51200 1721 64344 1563 50220 1545 47628 1355 61302 1265 59120 1144 59201 1120 60225 1110 64611 991 63917 957 58239 946 59138 849 64328 801 50122 715 65308 651 45421 511 49507 393 64409 307 64379 305 61434 222 64387 26 64107 15 51307 4 61426 4 64701 3

O maior nível salarial encontrado foi nas atividades de serviços financeiros, sendo as quatro primeiras ligadas a essas funções: Caixas Econômicas - R$ 44.607 / Bancos de Desenvolvimento - R$ 35.301 / Bancos de Investimento - R$ 26.945 / Holdings de Instituições Financeiras - R$ 24.730.

A maior parte do quadro de funcionários (15,8%) trabalha em atividades alimentícias. Sendo que 8,9%	dos trabalhadores brasileiros no setor terciário estão concetrados no Comércio Varejista de Mercadorias em Geral, com Predominância de Produtos Alimentícios - Hipermercados e Supermercados (Cnae 47113) e 6,9%	trabalham em Restaurantes e Outros Estabelecimentos de Serviços de Alimentação e Bebidas (Cnae 56112).




###Autoavaliação

Com esse estudo consegui observar as principais características do mercado de trabalho brasileiro, sendo possível responder as perguntas propostas. Para isso não precisei chegar até a camada Gold, com a camada Silver sendo o suficiente para atingir o objetivo. Também não foi necessário criar um modelo de dados em Esquema Estrela ou Snowflake, pois a tabela flat conseguiu responder as perguntas pretendidas no estudo.

In [0]:
%sql
SELECT 
    (SELECT CORR(sexo, remuneracao) FROM rais2022_silver) AS correlation_sexo,
    (SELECT CORR(cor, remuneracao) FROM rais2022_silver) AS correlation_cor,
    (SELECT CORR(faixa_etaria, remuneracao) FROM rais2022_silver) AS correlation_idade,
    (SELECT CORR(escolaridade, remuneracao) FROM rais2022_silver) AS correlation_escolaridade,
    (SELECT CORR(uf, remuneracao) FROM rais2022_silver) AS correlation_uf;

correlation_sexo correlation_cor correlation_idade correlation_escolaridade correlation_uf -0.027103351053910783 -0.05164146138749974 0.07156180731501754 0.15498704188918808 0.026432627425250318

Pela tabela de correlação entre os dados podemos perceber uma correlação negativa entre gênero e salário, mostrando que o fato de ser mulher (indicador 2) afeta negativamente a remuneração. Em relação a raça, o indicador também ficou abaixo de 0, confirmando que os indicadores maiores (parda, amarela, preta) tem influência negativa. No entanto, ambas as correlação são próximas a 0, representando que o impacto não é tão significativo quanto para as outras características.

Tanto a faixa etária, quando o estado de origem e a escolaridade apresentaram correlações positivas, sendo a da escolaridade a mais significativa. Por isso, podemos inferir que a escolaridade é a caraterística que apresenta maior impacto no nível salarial do trabalhador.

A correlação com as Cnaes não foi realizada pois elas não possuem uma ordem crescente lógica para analisar, sendo a influência de se ter um número de classe maior ou menos uma informação sem relevância.

Uma evolução desse estudo seria analisar as mesmas informações na segmentação de estado, para observar como cada estado se comporta em relação a contratação de mulheres, negros, pessoas mais velhas, com menor escolaridade ou quaisquer outras considerações que julgue relevantes.
Também há a possibilidade de analisar essas características dentro de uma cnae específica.